core/drivers/cbmem_console.c

```diff
@@ -18,12 +18,12 @@
 
 /*
  * Structures describing coreboot's in-memory descriptor tables. See
- * <coreboot>/src/commonlib/include/commonlib/coreboot_tables.h for
- * canonical implementation.
+ * https://github.com/coreboot/coreboot/blob/master/src/commonlib/include/commonlib/coreboot_tables.h
+ * for canonical implementation.
```

Comment:
s/canonical/Canonical/

Reply:
I don't think that should be capitalized since it's not the first word in a sentence (it's the adjective canonical, not the company name here).

Reply:
indeed :)>
my mistake.